particle-iot/particle-cli
# Feature/sc 124023/add spinner

## Description
This PR will add spinners to API calls in order to wait for the responses.




## How to Test
1. Pull down the branch: `git pull && git checkout feature/sc-124023/add-spinner`
2. Install dependencies: `npm i`
3. Run tests: `npm run test:ci` && `npm run test:e2e -- --grep "Logic Function Commands"`
4. Attempt to use the different commands, check the messages and outputs:
   * Listing for sandbox
   * Listing for org
   * Creating
   * Executing for sandbox
   * Executing for org
   * Deploying new logic function
   * Deploying existing logic function
   * Disabling/enabling
   * Deleting



**outcome**
* Tests should pass
* In every command you will see spinner messages indicating an specific step that is happening (when there is an api interaction)

## Related Issues / Discussions




## Completeness
